assesment.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded",() =>{
+    console.log("page loaded");
+    fetchMovies()
+    SelectMovie()
+
+})
+
+function fetchMovies(){
+    const url = "https://ghibliapi.herokuapp.com/films"
+    // axios.get(url).then((response)=>{
+    //     response.data
+    //     })
+    
+    fetch(url).then(response =>{
+        return response.json() 
+        //console.log(response)
+    })
+    .then(data => {
+        let pick = data
+        console.log(pick[0])
+        let selection = document.querySelector("select");
+        
+        for(movie in pick){
+            let title = document.createElement("option");
+            title.value = pick[movie].title;
+            title.innerText = pick[movie].title;
+            selection.appendChild(title);
+
+        }
+    })
+}
+
+function SelectMovie(){
+    
+    const url = "https://ghibliapi.herokuapp.com/films"   
+    fetch(url).then(response =>{
+        return response.json() 
+    })
+    .then(data => {
+        let pick = data
+        console.log(pick[0])
+        let selection = document.querySelector("select");
+        let form = document.querySelector("form")
+        
+        selection.addEventListener('change',()=>{
```

Comment:
It would have been good to break down the function `SelectMovie()` into two parts (functions) one that does the fetching of the info and other that takes care of displaying the single movie info. As I suggested earlier setting up the event lister or the select shouldn't be done here.